source/docs/software/commandbased/commands.rst

```diff
@@ -310,11 +310,11 @@ To wait until a certain condition becomes ``true``, the library offers the ``Com
 
 ### Control Algorithm Commands
 
-There are commands for various control setups:
+There are many different ways to implement these controllers in the command framework for various control setups:
 
-- ``PIDCommand`` uses a PID controller. For more info, see :ref:`docs/software/commandbased/pid-subsystems-commands:PIDCommand`.
+- ``PIDController`` can be integrated into a ``Subsystem``. For more info, see :ref:`docs/software/commandbased/pid-subsystems-commands:PID Control in Command-based`.
 
-- ``TrapezoidProfileCommand`` tracks a trapezoid motion profile. For more info, see :ref:`docs/software/commandbased/profile-subsystems-commands:TrapezoidProfileCommand`.
+- ``TrapezoidProfile`` tracks a trapezoid motion profile. For more info, see :doc:`/docs/software/commandbased/profile-subsystems-commands`.
```

Comment:
This section lists controller command classes, and should be removed once all the controller command classes are phased out. In the meantime, an admonition pointing to the other pages might be helpful (but I'm not entirely convinced it's needed); it shouldn't be an item in a list of command classes.